be/src/common/config.h

```diff
@@ -378,6 +378,17 @@ CONF_Int32(be_exit_after_disk_write_hang_second, "60");
 // turn off dictionary dictionary encoding. This only will detect first chunk
 // set to 1 means always use dictionary encoding
 CONF_Double(dictionary_encoding_ratio, "0.7");
+
+// Some data types use dictionary encoding, and this configuration is used to control
+// the size of dictionary pages. If you want a higher compression ratio, please increase
+// this configuration item, but be aware that excessively large values may lead to
+// performance degradation.
+CONF_Int32(dictionary_page_size, "1048576");
+
+// Just like dictionary_encoding_ratio, dictionary_encoding_ratio_for_non_string_column is used for
+// no-string column.
+CONF_Double(dictionary_encoding_ratio_for_non_string_column, "0");
```

Comment:
Can we reuse `dictionary_encoding_ratio` both for non string and string type?

Reply:
@luohaha I dont think we can reuse dictionary_encoding_ratio both for non string and string type. The reason is that the new item, dictionary_encoding_ratio_for_non_string_column, used for control whether enable/disable dict encoding for numerical types. Introduce a new item can minimize the intrusion of the new feature into the old behavior